components/CardTile.vue

```diff
@@ -34,6 +66,24 @@ export default {
     },
     list () {
       return this.$store.state.trelloLists.find(l => this.card.idList === l.id)
+    },
+    humanCertified () {
+      if (this.card.platform && this.card.platform.humanCertifications.length > 0) {
+        if (this.card.platform.humanCertifications[0].form.year) {
+          return true
+        }
+        return false
+      }
+      return false
+    },
+    publisherCertified () {
+      if (this.card.platform && this.card.platform.publisherCertifications.length > 0) {
+        if (this.card.platform.publisherCertifications[0].form.year) {
+          return true
+        }
+        return false
+      }
+      return false
```

Comment:
Deep nested prop without verification can lead to fatal errors. May be simplier to split this into several computed properties. (ex: platform => this.card.platform, pubCerts => this.platform.pubCerts)